packages/components/src/autocomplete/tests/chromatic/Autocomplete.chroma.jsx

```diff
@@ -0,0 +1,482 @@
+import { Autocomplete } from "@components/autocomplete";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong